packages/vue3-vite/src/defaults.ts

```diff
@@ -25,6 +25,8 @@ export const stylelintConfigVersion = '^29.0.0';
 export const tslibVersion = '^2.4.1';
 export const tsConfigVersion = '^7.0.0';
 export const unpluginVueComponentsVersion = '^0.22.12';
+export const nrwlJestVersion = '^15.3.3';
+export const typescriptVersion = '~4.8.2';
```

Comment:
I got these version from the `package.json` of `npx create-nx-workspace@latest`.

Reply:
Makes sense! But I think it's safe to bump typescript to the latest (4.8.4)